src/components/atoms/input.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Input(props) {
+  return (
+    <input type="checkbox"
+      value={props.value}
+      name={props.name}
+      onChange={props.updateSelection}
```

Comment:
Maybe keep props.updateSelection name consistent and call it `onChange`. And maybe this particular component isn't really needed since it's just passthrough props

Reply:
fixed